### DISCUSSION TITLE
@entangle does not work on the new version of alpine - 3.x

### DISCUSSION THREAD

**Discussion by patrickquijano** [livewire/livewire#3186] [7. Bugs (archived)]
Here are the screenshots of the errors.

![image](https://user-images.githubusercontent.com/6316829/123370759-f79ba600-d5b2-11eb-971b-e621a2eccb9e.png)
![image](https://user-images.githubusercontent.com/6316829/123370808-0f732a00-d5b3-11eb-810d-2897ad915288.png)
![image](https://user-images.githubusercontent.com/6316829/123371062-88728180-d5b3-11eb-8a05-b731b1bd35d4.png)

I have the latest version of Livewire. I am not sure what went wrong.

Does @entangle work on the new version of Alpine?

**Comment by joshhanley** [comment]
@patrickquijano can you please share some code, so we can try and replicate the issue? I've tested `@entangle` and haven't had any issues with it.

Thanks!

**Reply by patrickquijano** [reply]
Here's the component class:
```
<?php

namespace App\Http\Livewire;

use Livewire\Component;

class ResponseLanguage extends Component
{
    /**
     * The form open.
     *
     * @var bool
     */
    public $formOpen = false;

    /**
     * The response language.
     *
     * @var \App\Models\ResponseLanguage
     */
    public $language;

    /**
     * Renders the view.
     *
     * @return \Illuminate\View\View
     */
    public function render()
    {
        return view('livewire.response-language');
    }
}
```

Here's the component view:
```
<x-table-row theme="dark">
    <x-table-cell theme="dark">
        <x-table-cell-grid x-data="{ open: @entangle('formOpen') }">
            <template x-if="!open">
                @if (!is_null($language))
                    <span>
                        {{ $language->name }}
                    </span>
                @else
                    <x-button type="button" theme="light" value="Add" x-on:click="open = true" />
                @endif
            </template>

            <template x-if="open">
                <x-form class="gap-4 grid max-w-xs" wire:submit.prevent="save">
                    <x-form-group>
                        <x-label for="response_language_name" theme="dark" value="Name" />

                        <x-input type="text" theme="dark" required wire:model.debounce="language.name" />
                    </x-form-group>

                    <x-form-group actions>
                        <x-button type="submit" theme="primary" size="sm" value="Save" />

                        <x-button type="button" theme="danger" size="sm" value="Cancel" x-on:click="open = false" />
                    </x-form-group>
                </x-form>
            </template>

            @if (!is_null($language))
                <x-table-cell-actions>
                    <x-button theme="primary" size="xs" title="Edit" value="Edit" />

                    <x-button type="button" theme="danger" size="xs" title="Delete" value="Delete" wire:click="delete" />
                </x-table-cell-actions>
            @endif
        </x-table-cell-grid>
    </x-table-cell>
</x-table-row>
```

**Reply by joshhanley** [reply]
@patrickquijano thanks, that all looks ok! Can you tell me exactly which version of Alpine you are using?

Also any change you can try to test it without all of the blade components, maybe delete everything in the component view and just put a div in with the `x-data` and `@entangle` in it, and something to show/hide, and see if you can get it to run?

**Reply by patrickquijano** [reply]
@joshhanley Here's the Alpine version I am using:
![image](https://user-images.githubusercontent.com/6316829/123496915-2c643780-d65d-11eb-9968-8b5404159e3b.png)

Yes, just using the div works fine. I am not sure what's going on but I cannot use my blade component for this matter.

**Reply by joshhanley** [reply]
@patrickquijano ok thanks, version is good. 

Ok cool, good to know basic solution works. Are you able to work out which blade component it is that's causing the issue? If so, which one and can you share it?

**Reply by patrickquijano** [reply]
No, unfortunately.

This is what I have inside the table-cell-grid component:
```
<div {{ $attributes->class(['gap-1 grid']) }}>
    {!! $slot !!}
</div>
```

Nothing really special but I don't understand why it does not work.

**Reply by joshhanley** [reply]
@patrickquijano hmm bugger. Are you able to provide a sample repo that reproduces the issue, so I can pull it down to see if I can work out what is going on?

**Reply by calebporzio** [reply]
dInteresting, the `@` symbol in `@entangle` from your error message shouldn't be there - `@entangle` is a Laravel Blade directive and should get compiled away.

So something is going wrong in the Blade compilation step. This should have nothing to do with changing your Livewire version.

Maybe try clearing your compiled views? By running: `artisan view:clear`?
